soc/riscv/riscv-ite/it8xxx2/Kconfig.series

```diff
@@ -5,9 +5,11 @@ config SOC_SERIES_RISCV32_IT8XXX2
 	bool "ITE IT8XXX2 implementation"
 	#depends on RISCV
 	select COMPRESSED_ISA
-	select CPU_HAS_FPU
+	# TODO:
+	# Error of can't link soft-float modules with single-float modules.
+	# No library built with -mabi=ilp32f -march=rv32iafc?
```

Comment:
So if someone needs floating point, they have to fall back to buggy integer? =/
To workaround this, one would need to rebuild the toolchain?

Reply:
Yes. If we want to enable hardware floating point, we need to make toolchain's multilibs to
contain -march=rv32iafc -mabi=ilp32f combination. Or software floating point library will be used.